internal/database/sqlcommon/subscription_sql.go

```diff
@@ -122,6 +124,7 @@ func (s *SQLCommon) UpsertSubscription(ctx context.Context, subscription *core.S
 					subscription.Options,
 					subscription.Created,
 					subscription.Updated,
+					subscription.TLSConfigName,
```

Comment:
See earlier comments about this being part of the Webhook specific options - not sure we need a DB migration in that case.